_posts/2019-07-08-dask-custom-cupy-kernel.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: post
+title: Parallelizing Custom CuPy Kernels with Dask
+author: Peter Andreas Entschev
+tags: [Dask, CuPy]
+theme: twitter
+---
+{% include JB/setup %}
+
+
+Summary
+-------
+
+About three months ago, [Matthew Rocklin](https://twitter.com/mrocklin) wrote a
```

Comment:
```suggestion
About three months ago, we wrote a
```

Reply:
You always give nominal credit to everybody, why don't you want that for yourself?

Reply:
I can remove the link or change it to github profile if you prefer too.